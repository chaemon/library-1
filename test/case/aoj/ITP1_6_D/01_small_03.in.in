6 6
1 2 2 2 0 1
5 3 0 3 0 1
3 1 4 1 1 0
1 0 0 1 1 1
8 8 2 1 2 3
1 0 1 0 1 0
1
2
3
4
5
6
