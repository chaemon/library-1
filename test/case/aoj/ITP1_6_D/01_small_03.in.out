25
29
26
16
62
9
